playwright-e2e/dsm/pages/tablist/shared-learning-tab.ts

```diff
@@ -22,6 +22,6 @@ export default class SharedLearningTab extends TabBase {
   //Use screenshot comparison for the 'Upload Shared LEarnings Letter here...' section
 
   public async getUploadedFileCount(): Promise<number> {
-    return this.page.locator(`//app-files-table//table//tbody[@role='rowgroup']//tr`).count();
+    return this.page.locator(`//app-files-table//table//tbody[@role='rowgroup']//tr//td[contains(@class, 'mat-column-Name')]`).count();
```

Comment:
seemed to need additions to the xpath to find the files - but the change definitely makes it more brittle - will try to change to a less brittle locator